app/main.py

```diff
@@ -1,3 +1,72 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
```

Comment:
The type hint `int | float` is not valid in Python versions before 3.10. If you are using a version prior to 3.10, consider using `Union[int, float]` from the `typing` module instead.